code/client/src/sdk/mafia/ui/hud/c_hud_controller.h

```diff
@@ -1,9 +1,44 @@
 #pragma once
 
+#include "sdk/patterns.h"
+
+#include <utils/hooking/hooking.h>
```

Comment:
You probably don't need to re-import hooking here (already imported by pattern.h)